src/umpire/ResourceManager.cpp

```diff
@@ -32,6 +32,10 @@
 #include "umpire/resource/RocmPinnedMemoryResourceFactory.hpp"
 #endif
 
+#if defined(UMPIRE_ENABLE_NUMA)
+#include "umpire/resource/NUMAMemoryResourceFactory.hpp"
```

Comment:
Can you rename the file NumaMemoryResourceFactory. It's a bit nit-picky but I think it looks better.